src/terms/ewald.jl

```diff
@@ -33,6 +33,8 @@ function energy_ewald(model::Model{T}; kwargs...) where {T}
     energy_ewald(model.lattice, charges, model.positions; kwargs...)
 end
 
+# This could be factorised with Pairwise, but its use of `atom_types` would slow down this
```

Comment:
could be merged with ... also put an equivalent comment into the pairwise (if not there already)